### PR TITLE
add extra question to bug report to check the way of installing

### PR DESCRIPTION
**Pull request by mollux**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [x]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

add an extra question the the bug report template, so we know the way of installing.
This helps triaging bugs we can't e.g. reproduce on the dev installation of Mautic, as they are related to Composer (e.g. the locked version of dependencies)

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/BUG-REPORT.yml [hunk 1]
```diff
@@ -64,7 +77,7 @@ body:
     id: how_to_reproduce
     attributes:
        label: How can we reproduce this issue?
-       description: Explain us carefuly how we can reproduce the bug you faced.
+       description: Explain us carefully how we can reproduce the bug you faced.
```
Comment (RCheesley) [inline .github/ISSUE_TEMPLATE/BUG-REPORT.yml]:
```suggestion
       description: Explain to us carefully how we can reproduce the bug you faced.
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Looks good, just noticed a small typo/grammar issue!

**Review by escopecz — APPROVED** [review]
This will be very helpful. Thanks! 👍

**Review by RCheesley — APPROVED** [review]
🚀

**Pull request merged**